src/optimizer/statistics/selectivity_util.cpp

```diff
@@ -4,8 +4,62 @@
 #include "parser/expression_defs.h"
 
 namespace noisepage::optimizer {
+
+double SelectivityUtil::ComputeSelectivity(const TableStats &table_stats, const ValueCondition &condition) {
+  if (table_stats.GetNumRows() == 0) {
```

Comment:
I'm not actually sure if this should be 0 or 1. What's the selectivity of a column with no rows?